src/static/css/almanac.css

```
@@ -387,6 +386,38 @@ h2.header {
   font-size: 0.875rem;
   text-decoration: none;
   text-align: center;
+  box-shadow: 2px 2px 3px 3px lightgray;
+}
+
+.alt.btn.cta > svg {
+  margin-right: 0px;
+}
+
+.alt.btn.cta {
+  position: fixed;
+  padding: 10px 15px;
+  opacity: 0.9;
+  text-decoration: none;
+  z-index: 1;
+}
+
+.alt.btn.cta:nth-child(2) {
+  bottom: 8.5em;
+  right: 0.125em;
+}
+
+.alt.btn.cta:nth-child(3) {
+  bottom: 4.5em;
+  right: 0.125em;
+}
+
+.alt.btn.cta:nth-child(4) {
+  bottom: 0.6em;
+  right: 0.125em;
+}
+
+.alt.btn.cta > span:nth-child(2) {
+  margin-left: 0.35em;
```

This all seems a little verbose.

Can't we just use `.cta` for most of these, and maybe give real class names to the three ctas?

Also not sure about the name `cta`. Bit of a generic name. I wonder should we call it `chapter-cta`?

```suggestion
.btn.chapter-cta > svg {
  margin-right: 0px;
}

.chapter-cta {
  position: fixed;
  padding: 10px 15px;
  opacity: 0.9;
  text-decoration: none;
  z-index: 1;
}

.comment-cta {
  bottom: 8.5em;
  right: 0.125em;
}

.suggest-edit-cta {
  bottom: 4.5em;
  right: 0.125em;
}

.help-translate-cta {
  bottom: 0.6em;
  right: 0.125em;
}

.chapter-cta.> span:nth-child(2) {
  margin-left: 0.35em;
```

Still mulling over what we should do with that last one...